docs/backends.md

```diff
@@ -114,3 +114,13 @@ These are the parameters for AWS:
 AVP_TYPE: awssecretsmanager
 AWS_REGION: Your AWS Region (Optional: defaults to us-east-2)
 ```
+
+### GCP Secret Manager
+
+#### GCP Authentication
+Refer to the [Authentication Overview](https://cloud.google.com/docs/authentication) for Google Cloud APIs.
+
+These are the parameters for GCP:
+```
+AVP_TYPE: gcpsecretmanager
+```
```

Comment:
Could you add an example yaml to the GCP docs section? To show an example of what an annotation or inline path would look like?

Reply:
Sure! Will fix! :+1: